crates/router/src/connector/stripe/transformers.rs

```diff
@@ -1345,10 +1471,52 @@ impl TryFrom<&types::PaymentsAuthorizeRouterData> for PaymentIntentRequest {
             setup_mandate_details,
             off_session: item.request.off_session,
             setup_future_usage: item.request.setup_future_usage,
+            payment_method_types,
         })
     }
 }
 
+fn get_payment_method_type_for_saved_payment_method_payment(
+    item: &types::PaymentsAuthorizeRouterData,
+) -> Result<StripePaymentMethodType, error_stack::Report<errors::ConnectorError>> {
+    let stripe_payment_method_type = match item.recurring_mandate_payment_data.clone() {
+        Some(recurring_payment_method_data) => {
+            match recurring_payment_method_data.payment_method_type {
+                Some(enums::PaymentMethodType::Eps)
+                | Some(enums::PaymentMethodType::Giropay)
+                | Some(enums::PaymentMethodType::Przelewy24) => {
+                    Err(errors::ConnectorError::NotSupported {
+                        message: "Eps, Giropay, Przelewy24 Recurring payments ".into(),
+                        connector: "Stripe",
+                        payment_experience: "Recurring Payments".into(),
+                    }
+                    .into())
+                }
```

Comment:
```suggestion
```

This is not required, Is already handled in `StripePaymentMethodType::try_from(payment_method_type)`